### ISSUE TITLE
Letters ū Ū get distorted on chromium based browsers

### ISSUE THREAD

**Issue by pauliusws** [coreyhu/Urbanist#80]
Letters ū Ū get distorted on chromium based browsers when they have any character before or after it. The dash above the letter gets moved to the right side of the letter.
Issue can be reproduced on google fonts using any chromium based browser. 
No issue on firefox though.

**Comment by coreyhu** [comment]
I noticed the space is what offsets the dash/macron for some reason. Still investigating if this is a Chrome rendering issue and/or an issue with that glyph specifically.

**Comment by coreyhu** [comment]
Are you experiencing this on pages outside of Google Fonts?

**Comment by pauliusws** [comment]
Yes, I noticed this issue with the Urbanist font in a website I was
developing.

**Comment by phalkone** [comment]
I am noticing the same behaviour on chromium based browsers on all letters that contain the - accent and multiple letters. If you type ā on https://fonts.google.com/specimen/Urbanist it shows correct, but if you type Jā, you get the following:
<img width="190" alt="image" src="https://github.com/coreyhu/Urbanist/assets/640159/7449e2b6-7c01-44cc-931b-6ad878e23505">

**Comment by tykonu** [comment]
Same here. Could you check into it?

Looks fine on Firefox, the horizontal bars on 'ā' and 'i' and the like are messed up.
FF:
![image](https://github.com/coreyhu/Urbanist/assets/20248472/041193e8-e0cc-4d18-96b0-53009af4524b)

Chrome:
![image](https://github.com/coreyhu/Urbanist/assets/20248472/0f627d43-7086-4cf3-85ef-8663b6c69c00)


Italics versions seem fine on both.

**Comment by tykonu** [comment]
> I noticed the space is what offsets the dash/macron for some reason. Still investigating if this is a Chrome rendering issue and/or an issue with that glyph specifically.

It's not only the space, anything pushes it to the right.

It appears everywhere, no just Google fonts.

No space here:
![image](https://github.com/coreyhu/Urbanist/assets/20248472/56ceeb8a-0854-4bfa-b2ef-1fc68361b5d9)

**Comment by tykonu** [comment]
I see the issue is actually fixed in the repository. Could you add the new version to Google Fonts as well? Thanks!

**Comment by Meanski** [comment]
Could we please get this fix added to Google fonts, if the above comment is true.
